data/genre_descriptions_adults.csv

```diff
@@ -0,0 +1,8 @@
+Genre,Description
+Realism,"Art that represents subjects truthfully and without stylization, often focusing on everyday life scenes."
+Muralism,"Large-scale paintings typically created on walls or public buildings, often conveying social or political messages."
+Impressionism,A 19th-century art style focused on capturing light and movement with quick brushstrokes and vivid colors.
+Renaissance,"A cultural movement from the 14th–17th centuries focused on classical beauty, proportion, and religious or mythological subjects."
+Mannerism,"A late-Renaissance style known for its artificial elegance, exaggerated poses, and elongated forms."
+Symbolism,"An art movement using symbolic images and themes to express ideas, emotions, and imagination."
+Expressionism,"A modernist movement emphasizing emotional experience through bold colors, distorted forms, and subjective interpretation."
```

Comment:
We're missing genres from the data set it seems

Here is the full list:
https://github.com/TinaKgn/art_dna/blob/master/model/class_names.txt

I think we should have a description for each class